src/upload.js

```diff
@@ -35,25 +35,45 @@ const defaultUploadOptions = {
   tryFiles: undefined,
 };
 
+/**
+ * Uploads in-memory data to Skynet.
+ *
+ * @param {string|Buffer} data - The data to upload, either a string or raw bytes.
+ * @param {string} filename - The filename to use on Skynet.
+ * @param {Object} [customOptions={}] - Configuration options.
+ * @returns - The skylink.
+ */
+SkynetClient.prototype.uploadData = async function (data, filename, customOptions = {}) {
+  const opts = { ...defaultUploadOptions, ...this.customOptions, ...customOptions };
+
+  const sizeInBytes = data.length;
+
+  if (sizeInBytes < opts.largeFileSize) {
```

Comment:
this is kind of weird right, I can manipulate this through `customOptions`, shouldn't this be a constant?

Reply:
That's intended, below the threshold we use a single request and avoid the overhead of resumable uploads. The threshold is configurable by the dev.